### PR TITLE
SAK-49722 Sections student names should be in alphabetical order

### PR DESCRIPTION
**Pull request by victorGomollon**
[SAK-49722: Problem alphabetical order of the student names in the "Section info" tool](https://sakaiproject.atlassian.net/browse/SAK-49722)

### CODE REVIEW

**Review by jesusmmp — APPROVED** [review]

File: sections/sections-app/src/java/org/sakaiproject/tool/section/jsf/backingbean/EditManagersBean.java [hunk 1]
```diff
@@ -73,7 +75,8 @@ public class EditManagersBean extends CourseDependentBean implements Serializabl
 		public int compare(Object o1, Object o2) {
 			ParticipationRecord manager1 = (ParticipationRecord)o1;
 			ParticipationRecord manager2 = (ParticipationRecord)o2;
-			return manager1.getUser().getSortName().compareTo(manager2.getUser().getSortName());
+			Collator collator = Collator.getInstance(new Locale(System.getProperty("user.language"), StringUtils.isNotBlank(System.getProperty("user.region")) ? System.getProperty("user.region") : System.getProperty("user.country")));
```
Comment (ottenhoff) [inline sections/sections-app/src/java/org/sakaiproject/tool/section/jsf/backingbean/EditManagersBean.java:78]: is user.language and user.country set by default?

Comment (ern) [reply sections/sections-app/src/java/org/sakaiproject/tool/section/jsf/backingbean/EditManagersBean.java:78]: does using the locale that is returned from preferences, make sense?

Comment (jonespm) [reply sections/sections-app/src/java/org/sakaiproject/tool/section/jsf/backingbean/EditManagersBean.java:78]: It looks like there's already a few patterns that are in use in sections that should work:

` Locale locale = LocaleUtil.getLocale(FacesContext.getCurrentInstance());`

Or you should be able to use the `ResourceLoader`

```
Locale locale = new ResourceLoader().getLocale();
```

Comment (jesusmmp) [reply sections/sections-app/src/java/org/sakaiproject/tool/section/jsf/backingbean/EditManagersBean.java]: It works great with 

```
Collator collator = Collator.getInstance();
```

**Pull request merged**